shared/src/main/scala/mouse/feither.scala

```diff
@@ -42,6 +42,12 @@ final class FEitherOps[F[_], L, R](private val felr: F[Either[L, R]]) extends An
   def getOrElseIn[A >: R](right: => A)(implicit F: Functor[F]): F[A] =
     F.map(felr)(_.fold(_ => right, identity))
 
+  def getOrRaiseF[E](e: => E)(implicit F: MonadError[F, _ >: E]): F[R] =
```

Comment:
```suggestion
  def getOrThrow[E](e: => E)(implicit F: MonadError[F, _ >: E]): F[R] =
```

Reply:
I opted for `getOrRaiseF` for 3 main reasons: 
- have the `F` that means that works with the external effect so, it's aligned with the other ones
- `Throw` can be misleading and easily misunderstood `Raise`  let you immediately understand which method is calling under the hood 
- I've seen `getOrThrow` multiple times in different codebases, we can risk some naming collisions 

I'm open to changing it if I can't convince you :)

Reply:
Given `MonadError. raiseError` I accept the reasoning for why it's called `getOrRaiseF `

Reply:
If we speak about naming collisions, we should think only about `cats` and std. Only these libraries are our upstream. Or even if we look at `typelevel` libraries - `getOrThrow` is unique. Any other libraries are far away from our influence.

Probably having the `getOrRaiseIn` function is almost unreal, so I think we could simplify the function name a bit - `getOrRaise`. But yeah, I haven't much against the `raise`. So if you don't mind, can we remove the `F` suffix?

Reply:
From my side, I have nothing against it, we can remove the `F` suffix 👍